src/Lecture_3/Homework_3/Task_13.java

```diff
@@ -0,0 +1,61 @@
+package Lecture_3.Homework_3;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Task_13 {
+
+    public static void main(String[] args) {
+
+        Scanner sn = new Scanner(System.in);
+
+        System.out.println("Please enter the number of rows to form square two dimensional array");
+        int rows = sn.nextInt();
+        System.out.println("Please enter the number of columns to form square two dimensional array");
+        int columns = sn.nextInt();
+
+        if ((rows < 2) || (columns < 2)) {
+            System.out.println("Your array size values should be more than one");
+        } else {
+            if (rows != columns) {
+                System.out.println("Your array is not square.");
+            } else {
+                int [][] array = new int [rows][columns];
+
+                System.out.println("Please enter elements of array");
+
+                for (int i = 0; i < array.length; i++) {
+                    for (int j = 0; j < array[0].length; j++) {
+                        array[i][j] = sn.nextInt();
```

Comment:
Main diagonal is: arr[i][i]

Reply:
Sorry, I didn't understand this comment.